### PR TITLE
Test fixing duplicate fields

### PR DESCRIPTION
**Pull request by GuillaumeCSekoia**

### CODE REVIEW

**Comment by PierrickV** [comment]
@GuillaumeCSekoia: thank you for your work, could you update the CHANGELOG.md to describe your changes ?

File: AWS/aws-cloudtrail/tests/7602ff70-7e5f-42e9-86b2-36803df39183.json [hunk 1]
```diff
@@ -25,29 +25,7 @@
     },
     "@timestamp": "2022-08-31T09:48:47Z",
     "action": {
-      "name": "ModifyInstanceAttribute",
```
Comment (PierrickV) [inline AWS/aws-cloudtrail/tests/7602ff70-7e5f-42e9-86b2-36803df39183.json:28]: is this modification intended ? I cannot find the value duplicated in another field

Comment (GuillaumeCSekoia) [reply AWS/aws-cloudtrail/tests/7602ff70-7e5f-42e9-86b2-36803df39183.json:28]: it's in event.action

Comment (PierrickV) [reply AWS/aws-cloudtrail/tests/7602ff70-7e5f-42e9-86b2-36803df39183.json:28]: this field does not appear in the changelog

Comment (GuillaumeCSekoia) [reply AWS/aws-cloudtrail/tests/7602ff70-7e5f-42e9-86b2-36803df39183.json:28]: i have to put every field ?

Comment (PierrickV) [reply AWS/aws-cloudtrail/tests/7602ff70-7e5f-42e9-86b2-36803df39183.json:28]: Ah yes my bad

for the CHANGELOG.md its all good thank you

**Review by PierrickV — APPROVED** [review]
To avoid `prettier` issues I recommend that you install [pre-commit](https://pre-commit.com/#3-install-the-git-hook-scripts).

You will have some checks ran automatically before commiting

**Comment by GuillaumeCSekoia** [comment]
Thanks ! :) 
I let you merge if needed then I don't really know the process afterwards :)

**Comment by squioc** [comment]
I close this PR as we deployed a [similar one](https://github.com/SEKOIA-IO/intake-formats/pull/1078)

**Pull request closed without merging**